FormlyMapper/FormlyField/FileField.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class FileField extends FormlyField
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFieldType()
+    {
+        return 'file';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildFieldTypeConfiguration()
+    {
+        $this->formlyFieldConfiguration['type'] = $this->getFieldType();
```

Comment:
In formly they have the type as file instead of input? Because in HTML file is just type type of input.